src/boot/efi/util.h

```diff
@@ -84,12 +84,12 @@ static inline void *xmalloc_multiply(size_t size, size_t n) {
 /* Use malloc attribute as this never returns p like userspace realloc. */
 _malloc_ _alloc_(3) _returns_nonnull_ _warn_unused_result_
 static inline void *xrealloc(void *p, size_t old_size, size_t new_size) {
-        void *r = xmalloc(new_size);
+        void *new = xmalloc(new_size);
```

Comment:
uh, `new()` is a macro we define for memory allocations. I'd really like to avoid any such name clashes. I mean, sure C compilers distinguish a macro `foo` from a macro `foo()`, but relying on this is just ugly and confusing

Reply:
I changed it to `t`.